textile/api-docstrings.md

```diff
@@ -794,15 +794,19 @@ The summary entry for aggregated annotations that use the flag.v1 aggregation me
 |---|---|---|
 | total | number | The total number of clients who have published an annotation with this name (or type, depending on context). |
 | clientIds | string[] | A list of the clientIds of all clients who have published an annotation with this name (or type, depending on context). |
+| clipped | boolean | Indicates whether the clientIds array has been clipped (the total count is unaffected). |
 
 ## Class SummaryClientIdCounts
 
 The per-name value for the multiple.v1 aggregation method.
 
 | Property | Type | Description |
 |---|---|---|
-| total | number | The sum of the counts from all clients who have published an annotation with this name |
+| total | number | The sum of the counts from all clients who have published an annotation with this name. |
 | clientIds | { [key: string]: number } | A list of the clientIds of all clients who have published an annotation with this name, and the count each of them have contributed. |
+| totalUnidentified | number | The sum of the counts from annotations of unidentified clients. |
+| clipped | boolean | Indicates whether the clientIds map has been clipped (the total and totalClients counts are unaffected). |
```

Comment:
Perhaps be more specific and say that total/totalClientIds/totalUnidentified represent the values of the full-unclipped summary?

Reply:
Looks good to me!